DataReader/Classes/AppDelegate.swift

```diff
@@ -8,12 +8,23 @@
 
 import UIKit
 
-@UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
-
+    
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+        // Initialize Storyboard
+        
+        self.window = UIWindow(frame: UIScreen.main.bounds)
+        let storyboard = UIStoryboard(name: "Main", bundle: Bundle.main)
+        
+        // Instantiate Initial View Controller
+        if let viewController = storyboard.instantiateInitialViewController() as? ViewController {
```

Comment:
casting to ViewController isn't needed here

Reply:
done